pay.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+require('config.php');
+require('razorpay-php/Razorpay.php');
+session_start();
+
+// Create the Razorpay Order
+
+use Razorpay\Api\Api;
+
+$api = new Api($keyId, $keySecret);
+
+$orderData = [
+    'receipt'         => 3456,
+    'amount'          => 2000 * 100, // 2000 rupees in paise
+    'currency'        => 'INR',
+    'payment_capture' => 1 // auto capture 
```

Comment:
leave a link here to the orders API docs so people can figure out what this means